crates/bevy_color/src/testing.rs

```diff
@@ -0,0 +1,15 @@
+#[allow(unused_macros)]
```

Comment:
We might be able to replace these allows with a test cfg instead?

Reply:
Done